gpii/node_modules/flowManager/test/shared/BrowserChannelTestDefs.js

```diff
@@ -53,6 +53,15 @@ gpii.tests.flowManager.browserChannel.reportPlatform = function () {
     };
 };
 
+// A no-op to gracefully fail fetching the solutions registry from the source
+// code repository in order to use the canned registry set up for these tests.
+gpii.tests.flowManager.browserChannel.getRepositorySolutions = function (repositorySolutionsLoader) {
+    var togo = fluid.promise();
+    repositorySolutionsLoader.solutionsRegistry = null;
```

Comment:
This is another indication that having `repositorySolutionsLoader.solutionsRegistry` to track fetched SR may not be a good idea. Returning the SR in the promise returned by `get()` should be enough. Having 2 reaching points would confuse other developers when both points need to be taken care of when they'd like to do a similar overriding.

Reply:
RIght, and given I've change `RepositorySolutionsLoader` to return a promise to the`SRDS`, Ive re-worked this as appropriate.